node-src/io/getDNSResolveAgent.ts

```diff
@@ -7,7 +7,7 @@ export class DNSResolveAgent extends Agent {
       ...options,
       lookup(
         hostname: string,
-        lookupOptions: dns.LookupOneOptions,
+        _lookupOptions: dns.LookupOneOptions,
```

Comment:
`LookupFunction` does not have `lookupOptions`, but [does have `options`](https://nodejs.org/api/dns.html#dnslookuphostname-options-callback), this should probably be that?

Reply:
Yeah we can update that! They're positional arguments so it's not a huge deal what they're called.